### PR TITLE
Update code of conduct

### PR DESCRIPTION
**Pull request by Nasreen123**
We want to merge this on Wednesday night, and then send an email to attendees that will link to the CoC + also say that we will be asking people to add their pronoun to their name badge if they feel comfortable doing so.

### CODE REVIEW

File: _pages/coc.md [hunk 1]
```diff
@@ -8,14 +8,34 @@ permalink: /coc/
...
+All attendees, speakers, organizers and volunteers at Enthusiasticon are required to observe the following Code of Conduct. Organizers will enforce this code throughout the event.
 
-More information on this and how we are planning to apply the policy will follow as soon as we have defined it.
+-----------------
+
+Enthusiasticon is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, sexual orientation, disability, physical appearance, body size, race, religion, or anything else. We do not tolerate harassment of conference participants in any form.
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate for any conference venue, including talks.
+
+Please avoid making assumptions about people's pronouns. If you don't know what pronoun someone uses you can ask them or look at their name tag, where you may find their name and pronoun.  Otherwise please use their name rather than making an assumption about which pronoun to use.
```
Comment (Nasreen123) [inline _pages/coc.md]: We would like to find a nice video explaining this to people who are not used to the thinking about pronouns in this way.

File: _pages/coc.md [hunk 2]
```diff
@@ -8,14 +8,34 @@ permalink: /coc/
...
+Be kind to others. Do not insult or put down other attendees. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for Enthusiastion.
+
+Attendees violating these rules may be asked to leave the conference at the sole discretion of the conference organizers.
+
+Thank you for helping make this a welcoming, friendly event for all.
+
+Contact information
+-------------------
+
+Please email us at <coc@enthusiasticon.de> if you have any concerns about this Code of Conduct.
```
Comment (Nasreen123) [inline _pages/coc.md]: @zormit I got an undelivered email back from this address - can you put the correct one here so I can update it please

Comment (zormit) [reply _pages/coc.md]: It is correct though. Maybe we were to quick with sending the test, while the configuration change on the mailserver was not yet deployed. I only would suggest a change regarding markdown formatting:
```suggestion
Please email us at [coc@enthusiasticon.de](mailto:coc@enthusiasticon.de) if you have any concerns about this Code of Conduct.
```

File: _pages/coc.md (hunk 1 above)
Comment (zormit) [reply _pages/coc.md]: i liked [this one](https://www.youtube.com/watch?v=j_RZ672ii1E), explanation by first-time-reactions, which I like, as it's something people who are not used to might be able to relate.
For reference: Nasreen also found [this](https://lgbt.umd.edu/good-practices-names-and-pronouns), which we discussed and were not sure whether it's patronizing.

File: _pages/coc.md [hunk 3]
```diff
@@ -8,14 +8,34 @@ permalink: /coc/
...
-More information on this and how we are planning to apply the policy will follow as soon as we have defined it.
+-----------------
+
+Enthusiasticon is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, sexual orientation, disability, physical appearance, body size, race, religion, or anything else. We do not tolerate harassment of conference participants in any form.
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate for any conference venue, including talks.
+
+Please avoid making assumptions about people's pronouns. If you don't know what pronoun someone uses you can ask them or look at their name tag, where you may find their name and pronoun.  Otherwise please use their name rather than making an assumption about which pronoun to use.
+
+Be kind to others. Do not insult or put down other attendees. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for Enthusiastion.
```
Comment (zormit) [inline _pages/coc.md]:
```suggestion
Be kind to others. Do not insult or put down other attendees. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for EnthusiastiCon.
```

File: _pages/coc.md [hunk 4]
```diff
@@ -8,14 +8,34 @@ permalink: /coc/
...
+Attendees violating these rules may be asked to leave the conference at the sole discretion of the conference organizers.
+
+Thank you for helping make this a welcoming, friendly event for all.
+
+Contact information
+-------------------
+
+Please email us at <coc@enthusiasticon.de> if you have any concerns about this Code of Conduct.
+
+ During the conference we will have dedicated people who will be available to talk about anything relating to the code of conduct or attendees comfort and well being.  You can also email us during the conference if you prefer.
```
Comment (zormit) [inline _pages/coc.md]:
```suggestion
 During the conference we will have dedicated people who will be available to talk about anything relating to the code of conduct or attendees comfort and well being.  You can also email us during the conference if you prefer or use the following form (anonymous, via [formspree.io](https://formspree.io)):
 
<form action="https://formspree.io/coc@enthusiasticon.de" method="POST">

<label for="code of conduct violation">Code of Conduct violation </label>
<textarea type="text" name="code of conduct violation" rows="5" cols="80"></textarea>
<label for="_replyto">Your email (optional) </label>
<input type="email" name="_replyto">

<input type="submit" value="Send">
</form>  
```

File: _pages/coc.md (hunk 1 above)
Comment (hellerve) [reply _pages/coc.md]: I like the second one better. It’s definitely a little more forceful than the first, though the first isn’t as much of an explanation I find.

That said, I don’t have strong opinions for or against either of them.

Comment (Dzol) [reply _pages/coc.md]: I like the first even though it seems geared toward the work-place. Unfortunately I don't have strong preference between the two.

Comment (Nasreen123) [reply _pages/coc.md]: Would it be too much to use both?

Comment (hellerve) [reply _pages/coc.md]: I think that‘d be just fine! Good idea!

Comment (lelilia) [reply _pages/coc.md]: I strongly prefer the second video. The first one can also be included but it is not really explanatory and focuses on the eh so high burden on the others which I find weird to be honest.

File: _pages/coc.md (hunk 4 above)
Comment (Nasreen123) [reply _pages/coc.md]: :+1:

File: _pages/coc.md (hunk 3 above)
Comment (Nasreen123) [reply _pages/coc.md]: :+1:

File: _pages/coc.md (hunk 2 above)
Comment (Nasreen123) [reply _pages/coc.md]: :+1:

File: _pages/coc.md (hunk 1 above)
Comment (Nasreen123) [reply _pages/coc.md]: I went with only the second one.  The reason is that I just noticed the first uses 'preferred pronoun' in the title so I'd really rather not use it.

**Pull request merged**